beetsplug/replaygain.py

```diff
@@ -1378,20 +1463,13 @@ def imported(self, session, task):
     def commands(self):
         """Return the "replaygain" ui subcommand.
         """
-        def func(lib, opts, args):
```

Comment:
We generally keep these as nested functions in most plugins, just because they tend to be so simple.